cmd/selfupdate/help.go

```diff
@@ -0,0 +1,18 @@
+package selfupdate
+
+// Note: "|" will be replaced by backticks in the help string below
+var selfUpdateHelp string = `
+This command downloads the latest release of rclone and replaces
+the currently running binary. The download is verified with a hashsum.
```

Comment:
And cryptographically signed signature (if we get that working :-)

Reply:
And we got it working